scf/index.html

```diff
@@ -47,6 +47,9 @@
       <li>
 	<a href="{{page.root}}/scf/minutes.html">Minutes of Steering Committee Meetings</a>
       </li>
+      <li>
+      	<a href="https://github.com/swcarpentry/board/blob/master/subcommittees/subcommittees.md">Subcommittees</a>
```

Comment:
Could you replace with

~~~
      	<a href="{{github_url}}/board/blob/master/subcommittees/subcommittees.md">Subcommittees</a> 
~~~